k6ext/env.go

```diff
@@ -0,0 +1,40 @@
+package k6ext
+
+import (
+	"crypto/rand"
+	"math/big"
+	"os"
+	"strings"
+)
+
+// envResolve is defined as global var so we can overwrite it in
+// unit tests and therefore not manipulate the environment, which
+// allows us to run the test in parallel safely.
+var envResolve envResolver = os.LookupEnv //nolint:gochecknoglobals
+
+type envResolver func(key string) (string, bool)
+
+// IsRemoteBrowser returns true and the corresponding CDP
+// WS URL if this one is set through the K6_BROWSER_WS_URL
+// environment variable. Otherwise returns false.
+// If K6_BROWSER_WS_URL is set as a comma separated list of
+// URLs, this method returns a randomly chosen URL from the list.
+func IsRemoteBrowser() (wsURL string, isRemote bool) {
+	wsURL, isRemote = envResolve("K6_BROWSER_WS_URL")
+	if !isRemote {
+		return "", false
+	}
+	if strings.ContainsRune(wsURL, ',') {
+		parts := strings.Split(wsURL, ",")
+		// If last parts element is a void string,
+		// because WS URL contained an ending comma,
+		// remove it
+		if parts[len(parts)-1] == "" {
+			parts = parts[:len(parts)-1]
+		}
+		i, _ := rand.Int(rand.Reader, big.NewInt(int64(len(parts))))
+		wsURL = parts[i.Int64()]
+	}
+
+	return
```

Comment:
Thanks for considering my suggestion! Instead of saying what the rand does, I meant explaining "why" we put rand there: Selecting WS URLs in a round-robin fashion since we want to connect to a browser evenly.". Something like that. But it's fine. Don't worry about rechanging the code.

Btw, please add the return values to the last return statement:

```go
return wsURL, isRemote
```

[Here](https://github.com/golang/go/issues/21291) is some more food thought about the why.

Reply:
Got it! I was not sure whether to mention `round-robin` here, as that seems to be "away" of the competencies of this method, but I guess it's already part of its intention, so adding it to the doc comment.
In regards of the `return` statement. Thanks for the input! Was not aware of that. I see some discussions ongoing, so not sure what would be the most idiomatic way as of now, but changing it to be more explicit as we agree on this.
See [203c192](https://github.com/grafana/xk6-browser/pull/819/commits/203c192cb8511e1d914e76f1a53287ff05aaf82b)

Reply:
I believe, when you read a piece of code, it's better to know why it's there. So thanks, adding the comment.

The idiomatic way is to **never** use a naked return unless a function is teo or three lines of code.